packages/app/obojobo-document-engine/src/scripts/viewer/components/recommendation-component.js

```diff
@@ -0,0 +1,188 @@
+import './recommendation-component.scss'
+import React from 'react'
+import NavUtil from '../util/nav-util'
+import RecommendationUtil from '../util/recommendation-util'
+import GaugeChart from 'react-gauge-chart'
+
+const determineColor = (percent, settings) => {
+	for (const setting in settings) {
+		if (percent <= settings[setting].cutoff)
+			return settings[setting].color
+	}
+}
+
+const Summary = props => {
+	const thisPagePercent = 
+		props.moduleData.recommendationState.confidenceLevels[props.moduleData.recommendationState.currentPageID] / 100
+	const overallPercent = props.moduleData.recommendationState.overallConfidence / 100
+	const nwArcLength = 0.40
+	const goodArcLength = (1 - nwArcLength) / 3
```

Comment:
These types of values are better suited as constants, defined outside the component, like `const NW_ARC_LENGTH = 0.4`